Jenkinsfile

```diff
@@ -17,11 +18,24 @@
 
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
-//@Library(value='pipeline-lib@your_branch') _
+@Library(value='pipeline-lib@jvolivie/upload_fix') _
```

Comment:
this line needs to change back after https://github.com/daos-stack/pipeline-lib/pull/475 lands